public/app/plugins/datasource/loki/components/monaco-query-field/MonacoFieldWrapper.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import LokiLanguageProvider from '../../LanguageProvider';
+import { createLokiDatasource } from '../../mocks';
+
+import { MonacoQueryFieldWrapper, Props } from './MonacoQueryFieldWrapper';
+
+function renderComponent({
+  initialValue = '',
+  onChange = jest.fn(),
+  onRunQuery = jest.fn(),
+  runQueryOnBlur = false,
+}: Partial<Props> = {}) {
```

Comment:
1 thing I like to do is not export component `Props`, if we **only** need them in test file and use ComponentProps instead  `Partial<ComponentProps<typeof MonacoQueryFieldWrapper>>`. 

Here is an example:
https://github.com/grafana/grafana/blob/main/public/app/features/inspector/InspectDataTab.test.tsx#L13

Reply:
True, good suggestion.